### PR TITLE
DDF-2365: updated recommended Java version in DDF docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds a recommendation to use a Java version of 8u60 or higher.

#### Who is reviewing it?

@vinamartin @Lambeaux @jrnorth 

#### Choose 2 committers to review/merge the PR.

@shaundmorris
@stustison

#### How should this be tested?

review documentation

#### What are the relevant tickets?

[DDF-2365](https://codice.atlassian.net/browse/DDF-2365)

#### Screenshots (if appropriate)

screen shot 2016-08-05 at 9 11 19 am
screen shot 2016-08-05 at 9 15 57 am

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc [hunk 1]
```diff
@@ -10,6 +10,7 @@ These steps will demonstrate:
 ==== Prerequisites
 
 * Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The Recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or above.
```
Comment (jrnorth) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: I like "or later" (like you have in the other file) more than "or above". Also, "recommended" shouldn't be capitalized.

**Comment by Lambeaux** [comment]
:shipit:

**Comment by jrnorth** [comment]
LGTM apart from my comment.

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: fixed.

**Comment by shaundmorris** [comment]
👍

**Comment by shaundmorris** [comment]
@stustison @ricklarsen I'm thinking we should back port this documentation as well. Thoughts?

**Comment by ricklarsen** [comment]
I was going to ask about back-porting, actually. I say yes.

**Comment by ricklarsen** [comment]
Backport PR:

https://github.com/codice/ddf/pull/1104

**Pull request merged**